Extensions/Today/ButtonWithSublabel.swift

```diff
@@ -26,26 +26,32 @@ class ButtonWithSublabel: UIButton {
         self.titleLabel?.removeFromSuperview()
         addSubview(self.label)
         addSubview(self.subtitleLabel)
-
+        buttonImage.adjustsImageSizeForAccessibilityContentSizeCategory = true
+        buttonImage.contentMode = .scaleAspectFit
+        
         buttonImage.snp.makeConstraints { make in
-            make.centerY.left.equalTo(10)
-            make.width.equalTo(TodayUX.copyLinkImageWidth)
+            make.left.equalTo(10)
+//            make.top.equalTo(self.label.snp.top)
```

Comment:
nit: please remove comment

Reply:
@nishant2718 planning to remove any commented code, it's just there since this is a WIP. once I settle for the best approach, i will remove all commented code.